features/step_definitions/series_steps.rb

```diff
@@ -2,6 +2,12 @@
   visit series_url(Series.find_by(title: series))
 end
 
+Given "there are {int} works per page" do |amount|
+  stub_const("ArchiveConfig", OpenStruct.new(ArchiveConfig))
+  ArchiveConfig.ITEMS_PER_PAGE = amount.to_i
+  allow(Work).to receive(:per_page).and_return(amount)
```

Comment:
`@works` is an array. This means that `#paginate` calls `#per_page` on WillPaginate instead of the ActiveRecord, see the source here: https://rubydoc.info/gems/will_paginate/3.3.1/Array:paginate

So, this works:
```suggestion
  allow(WillPaginate).to receive(:per_page).and_return(amount)
```

The ArchiveConfig.ITEMS_PER_PAGE line can be removed. ([WillPaginate.per_page](https://github.com/otwcode/otwarchive/blob/master/config/initializers/gem-plugin_config/will_paginate_config.rb) is where that config is normally applied. But that's what we're stubbing here, so it's never used here).

Reply:
Looks like that did the trick!! Thank you!